src/Sulu/Bundle/CustomUrlBundle/Tests/Functional/Controller/CustomUrlControllerTest.php

```diff
@@ -612,8 +612,8 @@ public function testGet($data, $url)
         $this->assertArrayHasKey('creator', $responseData);
         $this->assertArrayHasKey('changer', $responseData);
 
-        $this->assertEquals(new \DateTime(), new \DateTime($responseData['created']), '', 2);
-        $this->assertEquals(new \DateTime(), new \DateTime($responseData['changed']), '', 2);
+        $this->assertLessThanOrEqual(new \DateTime(), new \DateTime($responseData['created']));
+        $this->assertLessThanOrEqual(new \DateTime(), new \DateTime($responseData['changed']));
```

Comment:
This is also true when the datetime has not been changed. You really should define a `$datetime` variable before the `$client->request()` call, and use it in a `assertGreaterThanOrEqual` call here.